streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorTopologyTest.java

```diff
@@ -222,9 +222,9 @@ public void shouldDriveGlobalStore() throws Exception {
     }
 
     @Test
-    public void testDrivingSimpleMultiSourceTopology() {
+    public void testDrivingSimpleMultiSourceTopology() throws Exception {
```

Comment:
Which part throws the exception? (same question for the other tests below). Could we use a specific exception, or is it broad?